eventdiscovery-sdk/src/main/java/com/schedjoules/eventdiscovery/eventdetails/UidEventDetails.java

```diff
@@ -46,10 +49,11 @@ public UidEventDetails(@NonNull String eventUid)
     @Override
     public void show(@NonNull Activity activity)
     {
-        Intent intent = new Intent(activity, MicroFragmentHostActivity.class);
-        Bundle nestedBundle = new Bundle(1);
-        nestedBundle.putParcelable("MicroFragment", new EventLoaderMicroFragment(mEventUid));
-        intent.putExtra("com.schedjoules.nestedExtras", nestedBundle);
-        activity.startActivity(intent);
+        activity.startActivity(
+                // TODO Use action intent instead?
```

Comment:
Not sure, but wasn't it the general suggested approach to use actions when possible. To keep things a bit more decoupled maybe. To be able to add a proxy activity for example if needed.
So the question is general in this sense. But I don't see any concrete need here.